proto/buf/alpha/registry/v1alpha1/push.proto

```diff
@@ -28,15 +28,15 @@ service PushService {
 message PushRequest {
   string owner = 1;
   string repository = 2;
-  string branch = 3;
+  string branch = 3 [deprecated = true];
```

Comment:
Will we eventually mark these as `reserved`, too?

Reply:
Yes! However throughout the cli version sometimes it'll send `main` as branch or as `track`, or none. We still want the server side to be able to read these to return informative error messages rather than push to the main track when a user did `buf push --track foo` because we can't read the track argument.